### DISCUSSION TITLE
微信新增发布能力接口，有更新计划吗？或是自己开发如何做插件？

### DISCUSSION THREAD

**Discussion by ezsky** [w7corp/easywechat#2630] [General]
接口文档如下：
https://developers.weixin.qq.com/doc/offiaccount/Publish/Get_publication_records.html

**Comment by overtrue** [comment]
6.x 的话本身就是支持所有接口的，5.x 的 client 也是支持原生调用的哈，看你用哪个版本。

**Reply by ezsky** [reply]
5.x 的 client的用法能否展开介绍一下 🙏。

**Comment by ljyljy0211** [comment]
一种自己写，也方便别人，还有种就是直接原生调用
https://easywechat.com/4.x/client.html
